BudgetSquirrel.Api/ClientApp/src/app/home/home/home.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+  selector: "bs-home",
+  template: `
+    <!-- Unauthenticated -->
+    <bs-splash-page *isAuthenticated="false"></bs-splash-page>
+
+    <!-- Authenticated -->
+    <bs-budgets *isAuthenticated="true"></bs-budgets>
```

Comment:
I don't see where "isAuthenticated" is coming from? How does this work?

Reply:
Ohhhh I see, this is that directive you created?

Reply:
Yeah it is a directive I created